kernel/packages/unity-interface/UnityInterface.ts

```diff
@@ -539,4 +531,25 @@ export class UnityInterface {
   }
 }
 
+export class ClientDebug {
+  unityInterface: UnityInterface
+
+  public constructor(unityInterface: UnityInterface) {
+    this.unityInterface = unityInterface
+  }
+
+  public DumpScenesLoadInfo() {
+    this.unityInterface.SendMessageToUnity('Main', 'DumpScenesLoadInfo')
+  }
+
+  public DumpRendererLockersInfo() {
+    this.unityInterface.SendMessageToUnity('Main', 'DumpRendererLockersInfo')
+  }
```

Comment:
Why using a class instead of exported top level funtions using the `let unityInterface`?

Reply:
I think exposing a `clientDebug` class is easier for discovering the exposed debug functions in there just by typing the class and `.` and seeing the console methods suggestions, instead of having to remember evey debug function we have for the client (right now that would be `DumpScenesLoadInfo()`, `DumpRendererLockersInfo()` and now `RunPerformanceMeterTool(durationInMilliseconds: number)`)